chat/irc.properties

```diff
@@ -80,24 +78,24 @@ message.whoisEntry=\ua0\ua0\ua0\ua0%1$S: %2$S
 message.unknownNick=„%S” nie jest znanym pseudonimem użytkownika.
 message.channelKeyAdded=Użytkownik %1$S zmienił hasło dla kanału na „%2$S”.
 message.channelKeyRemoved=Użytkownik %S usunął hasło kanału.
-message.banMasks=Użytkownicy łączący się z następujących adresów są niedopuszczani do „%S”:
-message.noBanMasks=Brak wykluczonych adresów dla „%S”.
-message.banMaskAdded=Użytkownicy łączący się z adresów pasujących do „%1$S” zostali wykluczeni przez %2$S.
-message.banMaskRemoved=Użytkownicy łączący się z adresów pasujących do „%1$S” nie będą dłużej niedopuszczani przez %2$S.
+message.banMasks=Użytkownicy łączący się z następujących adresów są zbanowani z „%S”:
```

Comment:
It was inconsistent before (wykluczać vs. niedopuszczać vs. even some existing instances of banować). "Banować" is a part of the language now ([PWN](http://sjp.pwn.pl/szukaj/banowa%C4%87.html)) and is, in my opinion, much easier to understand for the users of any chat program. If you're staunchly against it, then what do you propose instead? I guess we could use "wykluczać".